docs/rancher/07-run-murmuration-services/README.md

```diff
@@ -0,0 +1,164 @@
+# Configuring the Murmurations Network on an RKE2 Cluster
+
+## Introduction
+
+This guide outlines the necessary steps to configure DNS records, install cert-manager with Helm, update ingress configurations, create required secrets, deploy Murmurations services. By following this structured approach, you ensure that your RKE2 cluster is correctly set up to host the Murmurations services.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Step 1 - Configuring DNS Records](#step-1---configuring-dns-records)
+- [Step 2 - Switching Kubernetes Context](#step-2---switching-kubernetes-context)
+- [Step 3 - Installing Cert-Manager with Helm](#step-3---installing-cert-manager-with-helm)
+- [Step 4 - Updating Ingress Configuration](#step-4---updating-ingress-configuration)
+- [Step 5 - Creating Required Secrets](#step-5---creating-required-secrets)
+- [Step 6 - Deploying the Services](#step-6---deploying-the-services)
+- [Step 7 - Checking the Deployment](#step-7---checking-the-deployment)
+- [Conclusion](#conclusion)
+
+## Prerequisites
+
+Before you start, ensure you have:
+
+- Administrative access to your RKE2 cluster.
+- `kubectl` and `helm` tools installed and configured on your local computer.
+- A GitHub account for creating personal access tokens.
+
+## Step 1 - Configuring DNS Records
+
+Configure DNS A records for your services by pointing them to your load balancer URL created in the [Setup Load Balancer Tutorial](../05-setup-lb/README.md). Example DNS configurations:
+
+```bash
+my-index.murmurations.network C <load_balancer_url>
+my-library.murmurations.network C <load_balancer_url>
+my-data-proxy.murmurations.network C <load_balancer_url>
+```
+
+## Step 2 - Switching Kubernetes Context
+
+Switch to the Kubernetes context for your cluster where the Murmurations services will be deployed:
+
+```bash
+kubectl config use-context <context_name>
+```
+
+Replace `<context_name>` with your cluster's context name.
+
+## Step 3 - Installing Cert-Manager with Helm
+
+Install `cert-manager` in your cluster to manage certificates automatically:
+
+```bash
+helm install cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --set installCRDs=true
+```
+
+## Step 4 - Updating Ingress Configuration
+
+In this step, you'll update the [ingress.yaml](../../../charts/murmurations/charts/ingress/templates/ingress/ingress.yaml) configuration to correctly route traffic to your deployed services. Specifically, you'll replace the default URLs found in the ingress.yaml file with the custom URLs you've designated for your Murmurations services in Step 1.
```

Comment:
We also need to configure the issuer.yaml, as the email for the cert-manager requires modification.